### PR TITLE
SEO-175118-.NET-MAUI-ListView-Interlink

### PR DESCRIPTION
**Pull request by SellineRitaAnyangoOchieng**
Hi @Girija-Rajendran ,
|Title|Description|
|--|--|
|Task Category| .NET MAUI ListView|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|Interlink PR link| Interlink -https://github.com/syncfusion-content/maui-docs/pull/1396
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175118
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-ListView---Money-Keywords.aspx

 

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui listview|10-100 |3FT|To improve our FT experience |added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed. |
|.net maui listview example|10-100 |3FT|To move our FT ranking to 1st position | demo link added|
|.net maui listview binding |10-100|3FT|To improve our FT experience | improved on page changes|
|.net maui listview grouping|0-10 |Omit23FT To remove the page from omitted result and improve page experience| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|
|.net maui listview header|0-10 |Omit - 36FT| To remove the page from omitted result and improve page experience |added corresponding documentation links, arranged images in zigzag form|

 

Worked on 18 keywords in the .NET MAUI ListView and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-ListView---Money-Keywords.aspx

### CODE REVIEW

File: MAUI/ListView/getting-started.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Getting Started with .NET Maui ListView (SfListView)
 
-This section provides a quick overview of how to get started with the .NET Maui ListView (SfListView) for Maui. Walk-through the entire process of creating the real world SfListView.
+This section provides a quick overview of how to get started with the [.NET Maui ListView](https://www.syncfusion.com/maui-controls/maui-listview) (SfListView) for Maui. Walk-through the entire process of creating the real world SfListView.
```
Comment (Sharly-Atieno) [inline MAUI/ListView/getting-started.md:12]: Don't interlink here, instead you can combine both FT and example in N> section.

File: MAUI/ListView/overview.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Overview of .NET MAUI ListView (SfListView)
 
-The Syncfusion .NET MAUI ListView renders set of data items using Maui views or custom templates. Data can easily be grouped, sorted, and filtered.
+The Syncfusion [.NET Maui ListView](https://www.syncfusion.com/maui-controls/maui-listview) renders set of data items using Maui views or custom templates. Data can easily be grouped, sorted, and filtered.
```
Comment (Sharly-Atieno) [inline MAUI/ListView/overview.md:12]: .NET MAUI -capitalize

**Review by Sharly-Atieno — CHANGES_REQUESTED** [review]

File: MAUI/ListView/getting-started.md (hunk 1 above)
Comment (SellineRitaAnyangoOchieng) [reply MAUI/ListView/getting-started.md]: Resolved

File: MAUI/ListView/overview.md (hunk 2 above)
Comment (SellineRitaAnyangoOchieng) [reply MAUI/ListView/overview.md:12]: Updated

**Review by Sharly-Atieno — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**